examples/audio/ctc_asr.py

```diff
@@ -244,16 +249,74 @@ def encode_single_sample(wav_file, label):
 """
 
 
+# Reference: https://github.com/keras-team/keras/blob/ec67b760ba25e1ccc392d288f7d8c6e9e153eea2/keras/legacy/backend.py#L674-L711
+def ctc_label_dense_to_sparse(labels, label_lengths):
```

Comment:
Rather than rewriting this code, you can just use the built-in Keras 3 loss function `keras.losses.CTC`. I expect it will also enable the code example to run with all backends.

Reply:
Thanks for the feedback 👍
After removing the legacy code we still have some references to `tf` in the example and I'm not sure this can be made backend-agnostic.
Please let me know if I should substitute the remaining `tf` references.